compass/ocean/tests/global_ocean/forward.py

```diff
@@ -195,13 +208,18 @@ def run(self):
         Run each step of the testcase
         """
         config = self.config
-        # get the these properties from the config options
         for step_name in self.steps_to_run:
             step = self.steps[step_name]
-            # get the these properties from the config options
-            step.cores = config.getint('global_ocean', 'forward_cores')
-            step.min_cores = config.getint('global_ocean', 'forward_min_cores')
-            step.threads = config.getint('global_ocean', 'forward_threads')
```

Comment:
Previously, this was replacing the core and thread counts for all test cases with the values from the config file.  This meant that both steps of the decomp test case were running on 4 cores, and both steps of the thread test were running with 1 thread.